MEMBER_TRAVEL_FUND.md

```diff
@@ -63,21 +66,40 @@ might not be approved because of available funds, a separate issue will be raise
 Generally speaking, however, members shouldn't need to worry about the status
 of the fund itself.
 
-**The following considerations are made for approval of the request:**
+Examples of prior travel requests include attending a TC39 meeting,
+[Node.js collaboration summits][], and [Code and Learn][] mentoring.
```

Comment:
Micro-nit: Either make it `TC39 meetings` or else `collaboration summit`. (In other words, both plural or both singular.)